hut-finder-utilities/README.md

```diff
@@ -0,0 +1,27 @@
+# `hut-finder-utilities`
+
+Contains all the required data to get things running, on the database side.
+
+As of now, it just initialises hut data.
+
+## How to compile and run
+From the `hut-finder-utilities` directory...
+```
+go mod tidy
+```
+```
+go build
+```
+```
+./hut-finder-utilities-go initialiseHuts
```

Comment:
Following the steps, the binary created is called `hut-finder-utilities`

Reply:
yeah sorry, that was while i was porting things over and forgot to change, will do that now